content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -96,6 +96,46 @@ gateways is [actively in development](/docs/setup/upgrade/gateways/) and is cons
     $ helm upgrade istio-base manifests/charts/base -n istio-system --skip-crds
     {{< /text >}}
 
+### Stable revision labels (experimental)
+
+{{< warning >}}
+Stable revision labels are only supported when updating Istio from and to Istio versions 1.10+.
```

Comment:
cc @ericvn This is supported in earlier versions with istioctl, but not with Helm. Hopefully this makes this clear.